tendrl/node_agent/node_sync/__init__.py

```diff
@@ -164,10 +153,39 @@ def run(self):
                 sync_cluster_contexts_thread.daemon = True
                 sync_cluster_contexts_thread.start()
                 sync_cluster_contexts_thread.join()
-
+            # Update node alert count
+            update_node_alert_count()
```

Comment:
Do you really want to do this every sync as an overwrite, or re-calculate only if node is deleted due to TTL? @r0h4n comments?

Reply:
Should be done only if NS.tendrl.objects.NodeAlertCounters.exists() is False

Reply:
i will change this